### DISCUSSION TITLE
JAX and Flax articles

### DISCUSSION THREAD

**Discussion by mwitiderrick** [google/flax#2289] [Show and tell]
1. [What is JAX?](https://www.machinelearningnuggets.com/what-is-jax/) 
2. [Image classification with JAX & Flax](https://www.machinelearningnuggets.com/image-classification-with-jax-flax/ ) 
3. [Distributed training with JAX & Flax](https://www.machinelearningnuggets.com/distributed-training-with-jax-and-flax/) 
4. [JAX loss functions](https://www.machinelearningnuggets.com/jax-loss-functions/)
5. [How to use TensorBoard in Flax](https://www.machinelearningnuggets.com/how-to-use-tensorboard-in-flax/)
6. [How to load datasets in JAX with TensorFlow](https://www.machinelearningnuggets.com/how-to-load-dataset-in-jax-with-tensorflow/)

**Comment by marcvanzee** [comment]
Thanks for sharing!

**Reply by mwitiderrick** [reply]
Updated with new articles
